doc/source/reference/routines.dtypes.rst

```diff
@@ -0,0 +1,11 @@
+.. _routines.dtypes:
+
+Data Type Classes
```

Comment:
```suggestion
Data type classes
```

Just for consistency with the immediately preceding "Data type routines" article in the refguide: https://output.circle-artifacts.com/output/job/79c7052e-9400-40b6-bfa6-fe40fe062940/artifacts/0/doc/build/html/reference/index.html